chore(compute): release 1.0.0

noahdietz opened the pull request:
Promote `compute` to `ga`/stable ahead of v1 release.

I used the `!` breaking change indicator to precipitate a major version bump.

codyoss commented:
I don't think this needs to be a breaking change for release please. If you edit the commit message and touch a file with a whitespace change in the compute dir release-please will take care of this for us and open a PR. https://github.com/googleapis/release-please#how-do-i-change-the-version-number

noahdietz commented:
> I don't think this needs to be a breaking change for release please. If you edit the commit message and touch a file with a whitespace change in the compute dir release-please will take care of this for us and open a PR. 

Updated! I removed the beta documentation as my "whitespace change" :) That's the only thing that would change with the `beta`-> `ga` switch anyways.
